src/service/gateway.proto

```diff
@@ -2,7 +2,6 @@ syntax = "proto3";
 
 package helium;
 
-import "blockchain_state_channel_v1.proto";
```

Comment:
Why are you removing this?

Reply:
`protoc` raised a warning that this import was unused.